transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -136,6 +146,8 @@ public EventInternal decorate(EventInternal eventInternal) {
         .addMetadata(KEY_TIMEZONE_OFFSET, getTzOffset())
         .addMetadata(KEY_NETWORK_TYPE, getNetTypeValue(networkInfo))
         .addMetadata(KEY_MOBILE_SUBTYPE, getNetSubtypeValue(networkInfo))
+        .addMetadata(KEY_COUNTRY, Locale.getDefault().getCountry())
```

Comment:
Just noticed that while we are at it we can also add `Locale#getLanguage()`, do you mind adding it as well?